arcgis-ios-sdk-samples/Layers/Apply unique values with alternate symbols/ApplyUniqueValuesAlternateSymbolsViewController.swift

```diff
@@ -0,0 +1,113 @@
+        return numberFormatter
+    }()
+    
+    /// Creates the unique values renderer for the feature layer.
+    func createUniqueValuesRenderer() {
+        // Create the default symbol.
+        let symbol = AGSSimpleMarkerSymbol(style: .triangle, color: .red, size: 30)
+        // Convert the symbol to a multi layer symbol.
+        let multiLayerSymbol = symbol.toMultilayerSymbol()
+        multiLayerSymbol.referenceProperties = AGSSymbolReferenceProperties(minScale: 5_000, maxScale: 0)
+        // Create alternate symbols for the unique value.
+        let alternateSymbols = createAlternateSymbols()
+        // Create a unique value with alternate symbols.
+        let uniqueValue = AGSUniqueValue(description: "unique values based on request type", label: "unique value", symbol: multiLayerSymbol, values: ["Damaged Property"], alternateSymbols: alternateSymbols)
+        // Create a unique values renderer.
+        let uniqueValueRenderer = AGSUniqueValueRenderer()
+        // Create and append the unique value.
+        uniqueValueRenderer.uniqueValues.append(uniqueValue)
+        // Set the field name.
+        uniqueValueRenderer.fieldNames = ["req_type"]
+        // Create and set the default symbol.
+        let defaultSymbol = AGSSimpleMarkerSymbol(style: .diamond, color: .purple, size: 15)
+        uniqueValueRenderer.defaultSymbol = defaultSymbol.toMultilayerSymbol()
+        
+        // Set the unique value renderer on the feature layer.
+        featureLayer.renderer = uniqueValueRenderer
+    }
+    
+    /// Create alternate symbols for the unique value renderer.
+    func createAlternateSymbols() -> [AGSMultilayerPointSymbol] {
+        // Create the alternate symbol for the mid range scale.
+        let alternateSymbol = AGSSimpleMarkerSymbol(style: .square, color: .blue, size: 30)
+        // Convert the symbol to a multilayer symbol.
+        let alternateSymbolMultilayer1 = alternateSymbol.toMultilayerSymbol()
+        // Set the reference properties.
+        alternateSymbolMultilayer1.referenceProperties = AGSSymbolReferenceProperties(minScale: 10_000, maxScale: 5_000)
+        
+        // Create the alternate symbol for the high range scale.
+        let alternateSymbol2 = AGSSimpleMarkerSymbol(style: .diamond, color: .yellow, size: 30)
+        // Convert the symbol to a multilayer symbol.
+        let alternateSymbolMultilayer2 = alternateSymbol2.toMultilayerSymbol()
+        // Set the reference properties.
+        alternateSymbolMultilayer2.referenceProperties = AGSSymbolReferenceProperties(minScale: 20_000, maxScale: 10_000)
+        // Return both alternate symbols.
+        return [alternateSymbolMultilayer1, alternateSymbolMultilayer2]
+    }
+    
+    /// Update the label to display the current scale.
+    func changeScaleLabel() {
+        let mapScale = scaleFormatter.string(from: mapView.mapScale as NSNumber)!
+        currentScaleLabel.text = "Current scale: 1:\(mapScale)"
```

Comment:
Phil may argue about not using string concatenation for localization concerns, but I'm OK with it here because it is a status bar to show debug info, rather than a user facing UI component that needs to be localizable.